test/optim_adadelta_test.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from test.generic.optim_test_util import TestOptimizer
+
+from classy_vision.optim.adam import Adadelta
+
+
+class TestAdadeltaOptimizer(TestOptimizer, unittest.Testcase):
+
```

Comment:
you will need to disable the momentum buffer tests here like in Adam

Reply:
done